group-owner.html

```diff
@@ -25,6 +25,14 @@
 
 <p>The server administrator must assign the lab PI as the group owner. Then as group owner, the PI can add other users to the group and, if desired, assign others in the lab as group owners as well.</p>
 
+<p>
+    New in OMERO 5.4.0, the server administrator can also create one or more
+    <a href="https://docs.openmicroscopy.org/latest/omero/sysadmins/restricted-admins.html">Restricted Administrators</a> 
+    to delegate management of groups and data to a <a href="https://docs.openmicroscopy.org/latest/omero/sysadmins/restricted-admins.html#term-group-and-data-organizer">Group and Data Organizer</a>,
+    allowing a trusted user such as an imaging facility manager to perform
+    these task across groups.
```

Comment:
``...these task accross groups.`` -> ``...these tasks accross groups.`` (plural for ``tasks``)